# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR ORGANIZATION
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"POT-Creation-Date: 2023-07-29 11:01+0330\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: pygettext.py 1.5\n"


#: helpers/client_functions.py:75 helpers/client_functions.py:280
#: helpers/client_functions.py:381 helpers/client_functions.py:522
#: helpers/client_functions.py:641 helpers/client_functions.py:678
#: helpers/client_functions.py:740 helpers/client_functions.py:783
#: helpers/client_functions.py:860 helpers/client_functions.py:927
#: helpers/client_functions.py:1066 helpers/client_functions.py:1123
#: helpers/client_functions.py:1208 helpers/client_functions.py:1287
#: helpers/client_functions.py:1384 helpers/client_functions.py:1435
#: helpers/client_functions.py:1651 helpers/client_functions.py:1707
#: helpers/client_functions.py:1752 helpers/client_functions.py:1830
msgid "join_channel"
msgstr "You must join following channels before you can use bot!"

#: helpers/client_functions.py:86 helpers/client_functions.py:292
#: helpers/client_functions.py:392 helpers/client_functions.py:531
#: helpers/client_functions.py:688 helpers/client_functions.py:1075
#: helpers/client_functions.py:1132 helpers/client_functions.py:1217
#: helpers/client_functions.py:1296 helpers/client_functions.py:1444
#: helpers/client_functions.py:1761 helpers/client_functions.py:1839
msgid "user_not_found"
msgstr "User Not Found. Restart the bot: /start"

#: helpers/client_functions.py:147
msgid "no_server_available"
msgstr "No more servers available for you."

#: helpers/client_functions.py:157 helpers/client_functions.py:314
#: helpers/client_functions.py:440 helpers/client_functions.py:660
#: helpers/client_functions.py:765 helpers/client_functions.py:820
#: helpers/client_functions.py:888 helpers/client_functions.py:943
#: helpers/client_functions.py:994 helpers/client_functions.py:1416
#: helpers/client_functions.py:1679 helpers/client_functions.py:1733
msgid "general_cancel"
msgstr "‚ùå Cancel"

#: helpers/client_functions.py:161 helpers/client_functions.py:419
msgid "select_server"
msgstr "*Pick a Server*"

#: helpers/client_functions.py:162 helpers/client_functions.py:420
msgid "servers_list"
msgstr "*===== Server List =====*"

#: helpers/client_functions.py:212
msgid "error_account_creation_api_conflict"
msgstr "Error - Failed to Create Account (Database/API Conflict)"

#: helpers/client_functions.py:224
msgid "error_failed_account_creations"
msgstr "Error - Failed to Create Account"

#: helpers/client_functions.py:231 helpers/client_functions.py:492
msgid "error_get_config_api_conflict"
msgstr "Failed to get your config (API/Database Conflict)"

#: helpers/client_functions.py:235 helpers/client_functions.py:247
msgid "account_ready"
msgstr "Your Account is ready. Import and Enjoy!"

#: helpers/client_functions.py:302 helpers/client_functions.py:318
#: helpers/client_functions.py:402 helpers/client_functions.py:540
msgid "get_an_account"
msgstr "You Do not have an Account Yet. Get Your Vmess First."

#: helpers/client_functions.py:345
msgid "error_get_account_status"
msgstr "Error - Failed to Get Account Status (Database/API Conflict)"

#: helpers/client_functions.py:351
msgid "unlimited"
msgstr "Unlimited"

#: helpers/client_functions.py:352
msgid "account_status"
msgstr "*===== Account Status =====*"

#: helpers/client_functions.py:353
msgid "username"
msgstr "Username"

#: helpers/client_functions.py:354
msgid "server_name"
msgstr "Server Name"

#: helpers/client_functions.py:355
msgid "active"
msgstr "Active"

#: helpers/client_functions.py:356
msgid "upload_download"
msgstr "Upload / Download"

#: helpers/client_functions.py:357
msgid "used/limit"
msgstr "Used / Limit"

#: helpers/client_functions.py:485
msgid "refreshed_vmess"
msgstr "Here it is your refreshed vmesss"

#: helpers/client_functions.py:539 helpers/client_functions.py:545
#: helpers/client_functions.py:559 helpers/client_functions.py:586
#: helpers/client_functions.py:594
msgid "your_user_id"
msgstr "Your UserID"

#: helpers/client_functions.py:560
msgid "join_org"
msgstr "You must join an organization first! (Please make a purchase)"

#: helpers/client_functions.py:587 helpers/client_functions.py:595
msgid "contact_support"
msgstr "Please contact support"

#: helpers/client_functions.py:587 helpers/client_functions.py:595
msgid "error_code"
msgstr "*Error code*"

#: helpers/client_functions.py:615
msgid "usage"
msgstr "Usage"

#: helpers/client_functions.py:616
msgid "usage_limit"
msgstr "Limit"

#: helpers/client_functions.py:657
msgid "send_ticket"
msgstr "Please send your ticket"

#: helpers/client_functions.py:720
msgid "thanks_ticket"
msgstr "Thanks for your ticket ü§çÔ∏è\nWe will catch up with you soon!"

#: helpers/client_functions.py:762
msgid "referal_code"
msgstr "Please send your *referral code*:\nIf you don't have any, please contact your provider or send a ticket in \"Support\" section in the /menu."

#: helpers/client_functions.py:811
msgid "choose_plan"
msgstr "Please choose one of these plans:\nNote that the prices may be different based on USD/Tron/Rial exchanges and transaction fees."


#: helpers/client_functions.py:938
msgid "transaction_receipt"
msgstr "Please send your transaction receipt (image or text)"

#: helpers/client_functions.py:938 helpers/client_functions.py:954
#: helpers/client_functions.py:1409 helpers/client_functions.py:1542
#: helpers/client_functions.py:1726
msgid "selected_plan"
msgstr "Selected Plan"

#: helpers/client_functions.py:940 helpers/client_functions.py:1728
msgid "card_number"
msgstr "Card Number"

#  helpers/client_functions.py: 657
#: helpers/client_functions.py:941 helpers/client_functions.py:954
#: helpers/client_functions.py:1412 helpers/client_functions.py:1667
#: helpers/client_functions.py:1729
msgid "cancel_to_abort"
msgstr "Click Cancel to abort."

#: helpers/client_functions.py:954
msgid "zarinpal_message"
msgstr "Please make the transaction in the link below"

#: helpers/client_functions.py:993
msgid "pay_now"
msgstr "üí∞ Pay now"

#: helpers/client_functions.py:1046 helpers/client_functions.py:1048
msgid "not_yet_verified_payment"
msgstr "Your payment has not verified yet.\nPlease pay and try again later"

#: helpers/client_functions.py:1107 helpers/client_functions.py:1193
#: helpers/client_functions.py:1269 helpers/client_functions.py:1357
#: helpers/client_functions.py:1815 helpers/client_functions.py:1891
msgid "thanks_for_payment"
msgstr "Thanks for your payment ü§çÔ∏è\n\nWe will check your payment reciept and activate your account soon!\n\nPlease send your provider a message if your account was not recharged after 4 hours"

#: helpers/client_functions.py:1410
msgid "send_token"
msgstr "Please send your tokens for amount of"

#: helpers/client_functions.py:1410
msgid "to_our_wallet"
msgstr "to our wallet"

#: helpers/client_functions.py:1410
msgid "tron"
msgstr "Tron"

#: helpers/client_functions.py:1411
msgid "crypto_wallet_link"
msgstr "or make your payment at"

#: helpers/client_functions.py:1412
msgid "send_transaction_id"
msgstr "Send your *Transaction ID* below"

#: helpers/client_functions.py:1493
msgid "verify_crypto_manually"
msgstr "Thanks for your payment ü§çÔ∏è\nVerify your transaction using the above button."

#: helpers/client_functions.py:1497
msgid "verify_crypto_manually2"
msgstr "Please click on the button below in order to verify your transaction"

#: helpers/client_functions.py:1501
msgid "check_crypto_button"
msgstr "Check Transaction Verification"

#: helpers/client_functions.py:1528
msgid "already_verified_crypto_payment"
msgstr "Your payment had been verfied before!"

#: helpers/client_functions.py:1541
msgid "failed_crypto_transaction"
msgstr "‚ùå Your transaction is failed!"

#: helpers/client_functions.py:1542
msgid "error_wrong_amount"
msgstr "Possibility of wrong transaction ID."

#: helpers/client_functions.py:1542
msgid "error_wrong_trans_id"
msgstr "Possibility of incorrect amount or destination wallet on this transaction ID and selected plan."

#: helpers/client_functions.py:1543
msgid "check_trans_manually_2"
msgstr "You can check your transaction manually here"

#: helpers/client_functions.py:1544
msgid "contact_us_in_advance"
msgstr "If the everything is correct please contact us at"

#: helpers/client_functions.py:1574
msgid "not_verified_payment"
msgstr "Your payment is not verfied yet, please try again in a few seconds."

#: helpers/client_functions.py:1605
msgid "verified_account"
msgstr "‚úÖ Your payment was verified!\nThe account has been charged for"

#: helpers/client_functions.py:1606
msgid "added_to_org"
msgstr "You are now added to the organization. Please join the following channel for announcements about servers and config"

#: helpers/client_functions.py:1606
msgid "group"
msgstr "group"

#: helpers/client_functions.py:1667
msgid "choose_plan2"
msgstr "Please choose one of these plans"

#: helpers/client_functions.py:1726
msgid "send_crypto_transaction_receipt"
msgstr "Please send your transaction receipt (image or text)"

msgid "GB"
msgstr "GB"

msgid "T"
msgstr "T"